src/vendors/monitor.ts

```diff
@@ -130,3 +151,51 @@ export function parseTransactionsList(transactionsList: OperationsRow[]): Transa
     return parsed
   })
 }
+
+function getSharePrice(pool: oasis.types.StakingSharePool) {
+  const balance = Number(oasis.quantity.toBigInt(pool.balance!)) / 10 ** 9
```

Comment:
Will refer question to @pro-wh, because he usually helps me with such stuff.  Can `getSharePrice` lose precision: parsing big int to number, or we are safe here, or we don't care about it here ?

Reply:
the share price is a rational quantity, and we've been using plain old number to represent those, approximately as you point out

Reply:
We should convert all such math into bignumber https://github.com/oasisprotocol/oasis-wallet-web/issues/436#issuecomment-982186445

even low values can cause ugliness that I would rather not solve by rounding